draft-ietf-tls-tls13.md

```diff
@@ -2183,8 +2183,10 @@ hidden by TCP.
 
 A small allowance for errors in clocks and variations in measurements
 is advisable.  However, any allowance also increases the opportunity
-for replay.  In this case, it is better to reject early data than to
-risk greater exposure to replay attacks.
+for replay.  In this case, it is better to reject early data and fall back
+to a full 1-RTT handshake than to risk greater exposure to replay attacks,
+so in common network topologies, a small allowance on the order of ten
+seconds is reasonable.
```

Comment:
At the present moment since we've started enforcing limits, so I can't look that up presently since any numbers over > 120s are biased. 

The numbers from 120s to 10s though are unbiased. Even after excluding clients whose absolute time [-10, 10] days we still get about 0.3% of 0rtt lost over a week. 

Also android is worse than ios for drift.